src/eml/main.ml

```diff
@@ -21,10 +21,16 @@ struct
   let workspace_path =
     ref ""
 
+  let reason_syntax =
+    ref false
+
   let options = Arg.align [
     "--workspace",
     Arg.Set_string workspace_path,
     "PATH Relative path to the Dune workspace for better locations";
+    "--syntax-reason",
+    Arg.Set reason_syntax,
+    "Whether the templater should emit Reason syntax after preprocessing the template";
```

Comment:
The help string should probably just start from "Emit," as this is not a boolean option that you can set either way ("whether"), but a flag that causes a specific behavior if present.